api/python/quilt3/packages.py

```diff
@@ -383,19 +383,22 @@ def meta(self):
 
     @classmethod
     @ApiTelemetry("package.install")
-    def install(cls, name, registry=None, top_hash=None, dest=None, dest_registry=None):
+    def install(cls, name, registry=None, top_hash=None, dest=None, dest_registry=None, *, path=None):
         """
         Installs a named package to the local registry and downloads its files.
 
         Args:
-            name(str): Name of package to install. It also can be passed as NAME/PATH,
+            name(str): Name of package to install. It also can be passed as NAME/PATH
+                (deprecated, use 'path' parameter instead),
```

Comment:
```suggestion
                (/PATH is deprecated, use 'path=' keyword instead),
```

Reply:
I think `keyword` is not quite accurate. See https://docs.python.org/3/reference/lexical_analysis.html#keywords. I think we should either say `use the 'path' parameter` or `pass it as 'path' keyword argument`. See https://docs.python.org/3/glossary.html#term-parameter and https://docs.python.org/3/glossary.html#term-argument. Which one makes more sense?

Reply:
I tired argument to cover both because its kwarg and positional? See if you like it. Otherwise go with your best estimate.